SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialStaticComponentView.cpp

```diff
@@ -101,7 +101,7 @@ void USpatialStaticComponentView::OnAddComponent(const Worker_AddComponentOp& Op
 		// Component is not hand written, but we still want to know the existence of it on this entity.
 		Data = nullptr;
 	}
-	EntityComponentMap.FindOrAdd(Op.entity_id).FindOrAdd(Op.data.component_id) = std::move(Data);
+	EntityComponentMap.FindOrAdd(Op.entity_id).FindOrAdd(Op.data.component_id) = MoveTemp(Data);
```

Comment:
I'd encourage this to be separated out into a different PR. This is 1) a case where a stricter compiler complained about a missing include (e.g. &lt;algorithm&gt;) 2) a case where a code style wasn't followed, .. std::move and most stl isn't prevelant in the Unreal codebase. As a separate PR hopefully it stands out as an example for others about knowing when code doesn't match the style.

Reply:
double checked who made that change. it was @neprune  and @danielimprobable sometime in 2018, so a pretty long time ago.
This is the only occurrence of using a STL function in the Unreal GDK, so I'm not too worried about it happening again.